mlflow/models/cli.py

```diff
@@ -82,8 +82,10 @@ def predict(model_uri, input_path, output_path, content_type, json_format, no_co
 @cli_args.MODEL_URI
 @click.option("--name", "-n", default="mlflow-pyfunc-servable",
               help="Name to use for built image")
+@click.option("--gunicorn_opts", "-g", default=None,
+              help="gunicorn command used to start gunicorn with customized configuration")
```

Comment:
Help is inconsistent with flag. Either it's a command or it's options to the gunicorn command. To be consistent with serve I think it's preferable to have this be options.

Reply:
Updated